lib/pbench/server/indexing_tarballs.py

```diff
@@ -364,7 +388,7 @@ def sigquit_handler(*args):
                         # `linkerrdest` for later retry.
                         controller_path = linksrc_dir.parent
 
-                        if tb_res == 0:
+                        if tb_res == error_code["OK"]:
```

Comment:
Comparing objects/user-defined types with `==` makes me a little squirmy...I always have to go look up what it means.  If this is really what you want, I would recommend providing an explicit method on the object to do it.  Alternatively, if you accept Dave's suggestion to statically define `error_code`, then you can use `is` instead of `==`, which makes things clear.

Reply:
fixed